apps/trading/e2e/tests/liquidity_provision/test_liquidity_provision.py

```diff
@@ -46,7 +60,35 @@ def test_liquidity_provision_amendment(continuous_market, vega: VegaService, pag
     )
     next_epoch(vega=vega)
     page.reload()
+    expect(page.get_by_test_id("supplied-stake")).to_have_text("Supplied stake1.00001 tDAI")
+    expect(page.get_by_test_id("liquidity-supplied")).to_have_text("Liquidity supplied 17.16%")
+    row = page.get_by_test_id("tab-myLP").locator(".ag-center-cols-container .ag-row").first
+    expect(row).to_contain_text(
+        "Active"
+    )
+@pytest.mark.skip("Waiting for the ability to cancel LP")
+@pytest.mark.usefixtures("page", "auth", "risk_accepted")
+def test_liquidity_provision_inactive(continuous_market, vega: VegaService, page: Page):
```

Comment:
This is an example where if I read the code I literally do not know what is happening without going to find the non-specific css locators. For instance 'menuitemradio', totally generic and I would wrap this stuff personally. This is an area when I think about lowering the barrier to entry, if this stuff is abstracted it is easier for someone else to pick up

Reply:
Created a helper for swapping keys